.travis.yml

```diff
@@ -82,55 +87,57 @@ matrix:
         # Establish updated toolchain as default
         - sudo unlink /usr/bin/gcc && sudo ln -s /usr/bin/gcc-4.8 /usr/bin/gcc
         - sudo unlink /usr/bin/g++ && sudo ln -s /usr/bin/g++-4.8 /usr/bin/g++
+      script:
+        - ./bootstrap.sh -e -t && cd build  && cmake -DUSE_SHARED_LIBS= .. && make -j4 VERBOSE=1  && sudo make test ARGS="-j2 --output-on-failure"
```

Comment:
Why can't we use cmake --build instead of make here (and in the next one) too?

Reply:
We can. i think this and your above comment are sloppy cases where I rebased across machines and didn't adequately resolve the merges. Thanks

Reply:
To answer my own question based on the Travis results: because Trusty has cmake 2.8 and that does not support it. :(

Reply:
Yeah this week I keep getting time sliced to other tickets/bugs and software projects so I had completely forgotten why and assumed it was the fault of the merge. Little did I know I just can't remember anything!